.dockerignore

```diff
@@ -0,0 +1,2 @@
+venv/
+__pycache__
```

Comment:
This is incorrect, since `COPY . .` will copy *everything* not in this file. This includes the `.env` file and any differently named virtual environment, and also the Dockerfile itself (which is the least impact).

A good idea would be to simply symlink this to `.gitignore` although I'm not sure how this would work out on Windows hosts, but at the very least, this should have a lot more entries.

Reply:
Can you take care of this?

Reply:
I'll probably get to it next week sometime. Though I'm not sure if I should wait for #1 to be closed (I'm actually not even sure what the scope of that issue is, or why it's still open)

Reply:
you already merged #1  much before into your branch anyway which means it has made it to this PR anyway so I think you can go ahead with whatever you are going to do.

Reply:
Alright. I've sent a PR doing it the quick and dirty way, but I don't see it breaking anything in this repository.

Reply:
merged